packages/react-instantsearch/src/components/RangeInput.js

```diff
@@ -6,8 +6,9 @@ import classNames from './classNames.js';
 
 const cx = classNames('RangeInput');
 
-class RangeInput extends Component {
+export class RawRangeInput extends Component {
```

Comment:
Is this just for testing purpose? What was the issue you have?

Reply:
Yes, just for testing purpose. I want to shallow render the component in order to test some lifecycle hook where the problem come from.

Reply:
Is using `mount` would have change anything here?

Reply:
I don't think so. Inside the test I use `setProps` to trigger the hook `willReceiveProps` and the method is only available on the root wrapper.